docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml

```
@@ -0,0 +1,253 @@
+      - question: |
+          I have a database table with a `DateCreated` column that defaults to SQL `Getdate()`. When I try to insert a new record by using [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)], the value gets set to `NULL`. I would expect it to be set to the database default.
+        answer: |
+          [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] handles this situation automatically for identity (auto-increment) and rowguidcol (database-generated GUID) and timestamp columns. In other cases, you should manually set <xref:System.Data.Linq.Mapping.ColumnAttribute.IsDbGenerated%2A>=`true` and <xref:System.Data.Linq.Mapping.ColumnAttribute.AutoSync%2A>=<xref:System.Data.Linq.Mapping.AutoSync.Always>/<xref:System.Data.Linq.Mapping.AutoSync.OnInsert>/<xref:System.Data.Linq.Mapping.AutoSync.OnUpdate> properties.
+          
+  - name: Multiple DataLoadOptions
+    questions:
+      - question: |
+          Can I specify additional load options without overwriting the first?
+        answer: |
+          Yes. The first is not overwritten, as in the following example:
+          
+          ```vb
+          Dim dlo As New DataLoadOptions()
+          dlo.LoadWith(Of Order)(Function(o As Order) o.Customer)
+          dlo.LoadWith(Of Order)(Function(o As Order) o.OrderDetails)
+          ```
+          
+          ```csharp
+          DataLoadOptions dlo = new DataLoadOptions();
+          dlo.LoadWith<Order>(o => o.Customer);
+          dlo.LoadWith<Order>(o => o.OrderDetails);
+          ```
+          
+  - name: Errors Using SQL Compact 3.5
+    questions:
+      - question: |
+          I get an error when I drag tables out of a SQL Server Compact 3.5 database.
+        answer: |
+          The Object Relational Designer does not support SQL Server Compact 3.5, although the [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] runtime does. In this situation, you must create your own entity classes and add the appropriate attributes.
+          
+  - name: Errors in Inheritance Relationships
+    questions:
+      - question: |
+          I used the toolbox inheritance shape in the Object Relational Designer to connect two entities, but I get errors.
+        answer: |
+          Creating the relationship is not enough. You must provide information such as the discriminator column, base class discriminator value, and derived class discriminator value.
+
+  - name: Provider Model
+    questions:
+      - question: |
+          Is a public provider model available?
+        answer: |
+          No public provider model is available. At this time, [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] supports SQL Server and SQL Server Compact 3.5 only.
+          
+  - name: SQL-Injection Attacks
+    questions:
+      - question: |
+          How is [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] protected from SQL-injection attacks?
```

```suggestion
          How is LINQ to SQL protected from SQL-injection attacks?
```

We should replace ALL occurrences of `[!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)]` with `LINQ to SQL`.